### PR TITLE
Fix(client): RFQ workflow for spot tile in analytics (graph) mode (ARTP-927 & 934)

### PR DESCRIPTION
**Pull request by zhouyuhe**
- adding rfq mode in analytic mode with zeplin design
- zeplin link: https://app.zeplin.io/project/5b4660dc3b1097135eadcc35/screen/5e205fcdff4f36bf3d89260c
- screenshots:
image
image
image
image
image
image

### CODE REVIEW

File: src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/styled.tsx [hunk 1]
```diff
@@ -40,6 +40,10 @@ export const TradeButton = styled.button<{ direction: Direction; priceAnnounced:
   cursor: pointer;
   border: none;
   outline: none;
+  width: 122px;
+  min-width: 122px;
+  height: 59px;
+  min-height: 59px;
```
Comment (rikoe) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/styled.tsx]: Do these fixed widths apply to all usages of the price button? It doesn't introduce any regressions anywhere else does it? Did you check responsive mode, i.e. if you resize the app?

File: src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/PriceControls.tsx [hunk 2]
```diff
@@ -127,6 +137,7 @@ const PriceControls: React.FC<Props> = ({
         priceMovementType={priceMovement}
         spread={spreadValue}
         show={showPriceMovement}
+        isAnalyticView={!!isAnalyticsView}
```
[the remark this answers is not in the document]
Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/PriceControls.tsx]: cool, let me check it

File: src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/styled.tsx (hunk 1 above)
Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/styled.tsx]: I checked responsive mode for web, but I will check the mobile mode now.
Also, I think @fsarda and I will create another branch to make these children components responsive

File: src/client/src/apps/MainRoute/widgets/spotTile/components/stories/PriveMovement.stories.tsx [hunk 3]
```diff
@@ -6,13 +6,15 @@ import { select } from '@storybook/addon-knobs'
 stories.add('Price movement', () => {
   const priceMovementType = select('Direction', { Up: 'Up', Down: 'Down' }, 'Up')
   const showPriceMovement = true
+  const isAnalyticView = true
```
Comment (Sharakai) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/stories/PriveMovement.stories.tsx]: Why not change this to a `boolean` story knob?
```suggestion
  const isAnalyticView = boolean("Analytic View", true)
```

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx [hunk 4]
```diff
@@ -35,9 +44,9 @@ const BookingPill = styled.div<{ disabled: boolean; altStyle: boolean; color: st
   `}
 `
 
-const BookingStatus = styled.span`
+const BookingStatus = styled.span<{ altStyle: boolean }>`
```
Comment (Sharakai) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: nit: Not loving the name `altStyle`. It gives no context to what it actually is.
Perhaps `large: boolean`, seeing as it makes the font larger? Not sure.

Comment (Sharakai) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: Same applies to the above `BookingPill` component

File: src/client/src/apps/MainRoute/widgets/spotTile/components/analyticsTile/AnalyticsTile.tsx [hunk 5]
```diff
@@ -73,18 +89,28 @@ class AnalyticsTile extends React.PureComponent<SpotTileProps> {
                 showResetButton={showResetButton}
                 disabled={inputDisabled}
               />
+              {showTimer && rfqTimeout !== null && rfqReceivedTime !== null && (
+                <RfqTimer
+                  onRejected={handleRfqRejected}
+                  receivedTime={rfqReceivedTime}
+                  timeout={rfqTimeout}
+                  isAnalyticsView={true}
```
Comment (Sharakai) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/analyticsTile/AnalyticsTile.tsx]: You can omit the `={true}` for React boolean props with a static value of `true`.
```suggestion
                  isAnalyticsView
```

File: src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/styled.tsx (hunk 1 above)
Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/PriceButton/styled.tsx]: > The `isAnalyticView` prop seems to be getting passed through a number of components, but only used at the bottom-level.
> Have you considered putting this into React Context, instead?

good suggestions, I will do it in react context

File: src/client/src/apps/MainRoute/widgets/spotTile/components/analyticsTile/AnalyticsTile.tsx [hunk 6]
```diff
@@ -60,7 +76,7 @@ class AnalyticsTile extends React.PureComponent<SpotTileProps> {
           />
           <AnalyticsTileContent>
             <GraphNotionalWrapper>
-              <LineChartWrapper>
+              <LineChartWrapper isTimerOn={!!isTimerOn}>
```
[the remark this answers is not in the document]
Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/analyticsTile/AnalyticsTile.tsx]: good spot

File: src/client/src/apps/MainRoute/widgets/spotTile/components/stories/PriveMovement.stories.tsx (hunk 3 above)
Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/stories/PriveMovement.stories.tsx]: good point, I will look into storybook and especially for story knobs

**Pull request merged**